log-analyser/grafana-provisioning/dashboards/jicofo.json

```diff
@@ -18,15 +18,257 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
-  "id": 1,
   "links": [],
   "liveNow": false,
   "panels": [
     {
       "datasource": {
         "type": "loki",
-        "uid": "b8130a28-4867-4668-917d-539c93852857"
+        "uid": "a4bdfb3e-762a-46e5-a79f-2e7bbe88d444"
```

Comment:
see https://community.grafana.com/t/should-provisioned-dashboards-have-datasource-uids/65463

Reply:
https://user-images.githubusercontent.com/37156449/209871533-d89e3515-026b-4faa-94ae-1217a5f34705.gif